src/app/public/modules/toast/toast.component.scss

```diff
@@ -2,42 +2,6 @@
 @import "~@skyux/theme/scss/_compat/mixins";
 @import "~@skyux/theme/scss/variables";
 
-.sky-toast {
```

Comment:
Where these CSS changes intentional?

Reply:
Yes; they fix the icon problem. However, we'll need to release Toasts before it will show up in Stackblitz.